docs/sites.yml

```diff
@@ -9859,6 +9859,17 @@
     - Agency
   built_by: App Design
   built_by_url: "https://appdesign.dev/"
+- title: Clinka
+  url: https://www.clinka.com.au/
+  main_url: https://www.clinka.com.au/
+  source_url:
```

Comment:
```suggestion
```

Reply:
Thanks and sorry I didn't catch that myself!

Reply:
Not  a problem!